### DISCUSSION TITLE
How to set KaTeX's options in vscode

### DISCUSSION THREAD

**Discussion by kompre** [KaTeX/KaTeX#3894] [Q&A]
Hi,

Vscode uses katex engine to render math expression written in markdown files or jupyter notebook (ipynb). I would like to set some global option for katex (for example set `fleqn` to `true`), but I am not able to find how or where to define this settings. The extension that manages markdown content, either built-in or markdown-all-in-one, allows only to define macros. 

Do you know how I can define global options for the katex engine in vscode? 

Not sure if this the proper place to ask, but I am getting hard time finding proper documentation on the vscode repo or elsewhere.

**Comment by edemaine** [comment]
I don't think this is built into VSCode, but rather a plugin. It might be:
* [LaTeX workshop](https://github.com/James-Yu/LaTeX-Workshop). Maybe ask there?
* [mdmath plugin](https://marketplace.visualstudio.com/items?itemName=goessner.mdmath). There seems to be a `Katexoptions` option for this one.

**Comment by kompre** [comment]
I've already checked those, but I discovered they work only on .md files. I need it to work on a ipynb, and from my testing seems only the builtin vscode extension has an effect on those.

**Reply by edemaine** [reply]
Sorry, I missed that you wanted to support ipynb files. I agree it's not obvious where KaTeX gets applied; probably in a Markdown renderer. Probably you need to open a discussion/issue in the VSCode to request an options setting for KaTeX.
